portfolio/src/main/webapp/index.html

```diff
@@ -158,10 +156,13 @@ <h1>malachi blenman
                             <textarea name="lname"></textarea>
                             <p>enter your comment</p>
                             <textarea name="comment"></textarea>
-                            <br/><br/>
+                            <br></br>
                             <input type="submit"/>
-                        </form>
-                    </div>
+                        </form><br></br>
+                        <p>comments</p>
+                        <button onclick = "FetchDispGreeting()">press to view comments</button>
+                        <div id="dataa">
+                        </div>
```

Comment:
Any reason for the second "a" here?

Reply:
Just gave it a random variable name, but I can change it!